cmd/lakefs-loadtest/cmd/entry.go

```diff
@@ -57,9 +58,10 @@ var entryCmd = &cobra.Command{
 
 		conf := config.NewConfig()
 		c, err := catalog.NewCataloger(catalog.Config{
-			Config: conf,
-			DB:     database,
-			LockDB: lockDB,
+			Config:  conf,
+			DB:      database,
+			LockDB:  lockDB,
+			Actions: actions.New(database),
```

Comment:
Actions isn't a must for `EntryCataloger`. You can handle the nil case in graveler and avoid passing it from places where it's irrelevant.